### PR TITLE
Info Button - Session of Buttons #164

### LINKED ISSUE #164: Info Button - Session of Buttons
Inline with our "session of Buttons", We would be building different kinds of buttons..

Design a Button that looks like the button in this image...(Info_Button)

Create a file "Your_Button_Name.html" in the modules/buttons folder.

Use simple HTML and CSS to create the button.
For some additional info, You should read the instructions.txt file in "buttons" folder.
Screenshot from 2023-04-14 12-57-06

### PR DESCRIPTION
**Pull request by royninja**
Closes #164 

## Task
 created the info button as said mentioned in the issue section, only using html and css
 
---
- [x] Create a new branch for every feature you add.

- [x] Push and create a pull request to the dev branch.

- [x] Do not push directly to main, dev or deploy branches!

- [x] Use snake_case for branch names.

- [x] Write descriptive commit messages.

- [x] Explain what you did in a few words in your pull request comment.

- [x] Never merge or close a PR by yourself - one person is solely responsible for handling this to avoid confusion!

### CODE REVIEW

File: modules/buttons/info.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        align-items: center;
+    }
+    
+</style>
+
+<body>
+    <div>
+        <button class="info">
+         <span class="icon">
+            <svg xmlns="http://www.w3.org/2000/svg" x="0px" y="0px"
```
Comment (NtemKenyor) [inline modules/buttons/info.html:23]: I like the fact that you used svg. that's cool and smart.

File: modules/buttons/info.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    }
+    .icon,.text {
+        vertical-align: middle;
+        display: inline-block;
+        align-items: center;
+    }
+    
+</style>
+
+<body>
```
Comment (NtemKenyor) [inline modules/buttons/info.html:19]: Do we really need the body tag?

File: modules/buttons/info.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                <g fill="#ffffff" fill-rule="nonzero" stroke="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none" font-size="none" text-anchor="none" style="mix-blend-mode: normal"><g transform="scale(5.33333,5.33333)"><path d="M24,4c-11.02793,0 -20,8.97207 -20,20c0,11.02793 8.97207,20 20,20c11.02793,0 20,-8.97207 20,-20c0,-11.02793 -8.97207,-20 -20,-20zM24,7c9.40662,0 17,7.59339 17,17c0,9.40661 -7.59338,17 -17,17c-9.40661,0 -17,-7.59339 -17,-17c0,-9.40661 7.59339,-17 17,-17zM24,14c-1.10457,0 -2,0.89543 -2,2c0,1.10457 0.89543,2 2,2c1.10457,0 2,-0.89543 2,-2c0,-1.10457 -0.89543,-2 -2,-2zM23.97656,20.97852c-0.82766,0.01293 -1.48843,0.69381 -1.47656,1.52148v11c-0.00765,0.54095 0.27656,1.04412 0.74381,1.31683c0.46725,0.27271 1.04514,0.27271 1.51238,0c0.46725,-0.27271 0.75146,-0.77588 0.74381,-1.31683v-11c0.00582,-0.40562 -0.15288,-0.7963 -0.43991,-1.08296c-0.28703,-0.28666 -0.67792,-0.44486 -1.08353,-0.43852z"></path></g></g>
+            </svg>
+         </span>
+         <span class="text">Info</span>
+        </button>
+    </div>
+</body>
```
Comment (NtemKenyor) [inline modules/buttons/info.html:33]: Finally, To increase the level of acceptance of your PR, Always help me with a screenshot so I don't have to create a branch or check out the entire stuff here. LOL. Nice one.

**Review by NtemKenyor — APPROVED** [review]
Nice Job Bro. Thank you for contributing to open source development.

File: modules/buttons/info.html (hunk 2 above)
Comment (royninja) [reply modules/buttons/info.html:19]: Not required, I was using vscode and body was required to view the live changes, anyways I can remove that too.

File: modules/buttons/info.html (hunk 3 above)
Comment (royninja) [reply modules/buttons/info.html:33]: sure.

File: modules/buttons/info.html (hunk 1 above)
Comment (royninja) [reply modules/buttons/info.html:23]: thanks

**Pull request merged**